controllers/storagecluster/reconcile.go

```diff
@@ -532,6 +532,17 @@ func (r *StorageClusterReconciler) reconcilePhases(
 		}
 	}
 
+	// For security of encrypted messages, When both encryption and compression are enabled,
+	// compression setting will be ignored and message will not be compressed.
+	// Refer https://docs.ceph.com/en/quincy/rados/configuration/msgr2/#confval-ms_compress_secure
+	networkSpec := instance.Spec.Network
+	if networkSpec != nil && networkSpec.Connections != nil &&
+		networkSpec.Connections.Encryption != nil && networkSpec.Connections.Encryption.Enabled &&
+		networkSpec.Connections.Compression != nil && networkSpec.Connections.Compression.Enabled {
+		r.Log.Info("Both in-transit encryption & compression are enabled. ")
+		r.Log.Info("To protect security of encrypted messages ceph will ignore the compression setting by default here.")
+	}
```

Comment:
@malayparida2000 Where is the compression setting ignored? I only see the logging added here.

Reply:
Is not that going to be handled in the rook?

Reply:
@travisn  Ceph will ignore the compression setting, see here https://docs.ceph.com/en/quincy/rados/configuration/msgr2/#confval-ms_compress_secure

Reply:
Ah I missed that, thanks